video/out/vo_gpu.c

```diff
@@ -173,6 +173,17 @@ static void get_and_update_ambient_lighting(struct gpu_priv *p)
     }
 }
 
+static void update_ra_ctx_options(struct vo *vo)
+{
+    struct gpu_priv *p = vo->priv;
+
+    /* Only the alpha option has any runtime toggle ability. */
+    int alpha_mode;
+    mp_read_option_raw(vo->global, "alpha", &m_option_type_choice, &alpha_mode);
```

Comment:
No new uses of this function.

Reply:
Oops. I could have spent 5 seconds actually reading the header.